src/main/kotlin/com/demonwav/mcdev/i18n/inspections/NoTranslationInspection.kt

```diff
@@ -0,0 +1,119 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n.inspections
+
+import com.demonwav.mcdev.i18n.I18nElementFactory
+import com.demonwav.mcdev.i18n.lang.gen.psi.I18nProperty
+import com.demonwav.mcdev.i18n.reference.I18nGotoModel
+import com.demonwav.mcdev.i18n.reference.I18nReference
+import com.demonwav.mcdev.i18n.translations.identifiers.LiteralTranslationIdentifier
+import com.intellij.codeInspection.InspectionManager
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemHighlightType
+import com.intellij.ide.util.gotoByName.ChooseByNamePopup
+import com.intellij.ide.util.gotoByName.ChooseByNamePopupComponent
+import com.intellij.openapi.application.ModalityState
+import com.intellij.openapi.command.WriteCommandAction
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.openapi.ui.Messages
+import com.intellij.psi.JavaPsiFacade
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiLiteralExpression
+import com.intellij.util.IncorrectOperationException
+import org.jetbrains.annotations.Nls
+
+class NoTranslationInspection : TranslationInspection() {
+    private val createQuickFix = CreateTranslationQuickFix()
```

Comment:
Unless you really need a separate instance for each inspection instance you can just make the two quick fixes singletons (by declaring them as `object`)